tests/Go/Console/Command/CacheWarmupCommandTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Go\Console\Command;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+class CacheWarmupCommandTest extends TestCase
+{
+    protected $aspectCacheDir = __DIR__.'/../../../Fixtures/project/var/cache/aspect';
+
+    public function setUp()
+    {
+        $filesystem = new Filesystem();
```

Comment:
I don't think that it is possible at all to do that in conjunction of how I did wrote these tests.

These tests (console commands) are written as functional, black box, tests. Console commands are executed as separate command line process. PHP will start a new thread, and I am getting only stdout from it. 

So I am not sure if that is even possible, if it is, I will have to look how to do that.

On the other hand,  I think that these tests are more safe then VFS because it is a real working environment and I do not depend on third party mock to assert that some code is working properly or not.